saltlint/rules/JinjaPillarGrainsGetFormatRule.py

```diff
@@ -0,0 +1,22 @@
+# -*- coding: utf-8 -*-
+# Copyright (c) 2016, Will Thames and contributors
+# Copyright (c) 2018, Ansible Project
+# Modified work Copyright (c) 2019 Jeffrey Bouter
+
+from saltlint.linter import SaltLintRule
+import re
+
+
+class JinjaPillarGrainsGetFormatRule(SaltLintRule):
+    id = '211'
+    shortdesc = 'pillar.get or grains.get should be formatted differently'
+    description = "pillar.get and grains.get should always be formatted" \
+                  " like salt['pillar.get']('item') or grains['item1']"
+    severity = 'HIGH'
+    tags = ['formatting']
```

Comment:
By now there are several Jinja checks, so we might want to introduce a new tag called `jinja` to group them:

```suggestion
    tags = ['formatting', 'jinja']
```

Reply:
The `jinja` tag has been added